portfolio/src/main/webapp/script.js

```diff
@@ -221,6 +221,7 @@ function convertToDateTime(timestamp) {
   var date = new Date(timestamp);
   var dateFormatted = date.toLocaleDateString(); 
   var today = new Date();
+  // Check whether the date of the given timestamp is the same as today; if so, return number of hours ago.
```

Comment:
Instead of reading out the code, I would say something like:
"Use time in hours if timestamp is for the same day" (you can say it how you prefer)

Reply:
Got it.